recipes-domx/meta-xt-images-domx/recipes-extended/xen/xen-4.10-rocko.inc

```diff
@@ -1,71 +1,10 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
-
+require xen-rocko.inc
```

Comment:
this wants to be xen-common.inc, not rocko

Reply:
with this fixed:
`Reviewed-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>`